docs/how_to/kubernetes.md

```diff
@@ -3,7 +3,7 @@
 **Please note:** If possible _[PLEASE USE THE OPERATOR](https://operator.m3db.io/)_ to deploy to Kubernetes if you
 can. It is a considerly more streamlined setup.
 
-The [operator](https://operator.m3db.io/) leverages [custom resource definitions](https://v1-10.docs.kubernetes.io/docs/concepts/api-extension/custom-resources/)
+The [operator](https://operator.m3db.io/) leverages [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
```

Comment:
Should we keep these and just update the version?

Reply:
They've completely moved it unfortunately, I can't get access to any versioned docs.